### DISCUSSION TITLE
AS4 logs for peppol statistics

### DISCUSSION THREAD

**Discussion by runholen** [OxalisCommunity/Oxalis-AS4#239] [General]
Hi. We are a certified peppol endpoint. Unfortunately, the person that installed oxalis with AS4 has left the company, so I was a bit surprised when I just found out that we have to start sending extended statistics to peppol in february, as I had not heard of this before now.
Trying to build this statistics, I cannot seem to find logs for the incoming and outgoing AS4 messages.
For incoming, I get receipts in /opt/oxalis/var/IN 
The statistic documentation seems to refer to eg eb:UserMessage but I cannot find this in the .dat files in IN.
I can in these files also only find my own CNOxxxxxx-number (servicepoint nr), not the senders number. And outgoing traffic I cannot find any log files for at all. So where do I find a log of incoming and outgoing AS4-messages?

**Comment by aaron-kumar** [comment]
@runholen : Detailed documentation related to Peppol reporting is provided at : https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting. Since this is Not bug so changing it to discussion
